component/org.wso2.carbon.identity.api.otp.service.emailotp/src/gen/java/org/wso2/carbon/identity/api/otp/service/emailotp/dto/OTPValidationFailureReason.java

```diff
@@ -0,0 +1,142 @@
+    private String description;
+
+    /**
+    **/
+    public OTPValidationFailureReason code(String code) {
+
+        this.code = code;
+        return this;
+    }
+    
+    @ApiModelProperty(example = "EMAIL-60006", value = "")
+    @JsonProperty("code")
+    @Valid
+    public String getCode() {
+        return code;
+    }
+    public void setCode(String code) {
+        this.code = code;
+    }
+
+    /**
+    * OTP validation failure reason.
+    **/
+    public OTPValidationFailureReason message(String message) {
+
+        this.message = message;
+        return this;
+    }
+    
+    @ApiModelProperty(example = "Expired OTP.", value = "OTP validation failure reason.")
+    @JsonProperty("message")
+    @Valid
+    public String getMessage() {
+        return message;
+    }
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    /**
+    * OTP validation failure reason description.
+    **/
+    public OTPValidationFailureReason description(String description) {
+
+        this.description = description;
+        return this;
+    }
+    
+    @ApiModelProperty(example = "Expired OTP provided for the user Id: 8b1cc9c4-b671-448a-a334-5afe838a4a3b.", value = "OTP validation failure reason description.")
+    @JsonProperty("description")
+    @Valid
+    public String getDescription() {
+        return description;
+    }
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+
+
+    @Override
+    public boolean equals(java.lang.Object o) {
+
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        OTPValidationFailureReason otPValidationFailureReason = (OTPValidationFailureReason) o;
+        return Objects.equals(this.code, otPValidationFailureReason.code) &&
+            Objects.equals(this.message, otPValidationFailureReason.message) &&
+            Objects.equals(this.description, otPValidationFailureReason.description);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(code, message, description);
+    }
+
+    @Override
+    public String toString() {
+
+        StringBuilder sb = new StringBuilder();
+        sb.append("class OTPValidationFailureReason {\n");
+        
+        sb.append("    code: ").append(toIndentedString(code)).append("\n");
+        sb.append("    message: ").append(toIndentedString(message)).append("\n");
+        sb.append("    description: ").append(toIndentedString(description)).append("\n");
+        sb.append("}");
+        return sb.toString();
+    }
+
+    /**
+    * Convert the given object to string with each line indented by 4 spaces
+    * (except the first line).
+    */
+    private String toIndentedString(java.lang.Object o) {
+
+        if (o == null) {
+            return "null";
+        }
+        return o.toString().replace("\n", "\n");
+    }
+}
```

Comment:
Since this method is reused in other classes, let's have a Util class and include this there and use that

Reply:
`auto-generated` files.